script/core/README.md

```diff
@@ -0,0 +1,56 @@
+# Deployment
+
+1. Install dependencies by running `bun install`.
+2. Compile contracts by running `bun smock && bun compile`.
+3. Fill the necessary environment variables in `.env` file. You can use `.env.example` as a template.
```

Comment:
Can we update the `.env.example` and separate the deployment variables by a comment, so people can find easy which ones they have to fill

Reply:
Like so? https://github.com/allo-protocol/allo-v2.1/pull/3/commits/b4603cc234f7a4f0e09d793e25a55e803f21f359